tests/simpleKVBC/TesterReplica/setup.cpp

```diff
@@ -106,8 +116,10 @@ std::unique_ptr<TestSetup> TestSetup::ParseArgs(int argc, char** argv) {
   // This allows more concurrency and only affects known ids in the
   // communication classes.
   replicaConfig.numOfClientProxies = 100;
-  replicaConfig.autoViewChangeEnabled = rp.viewChangeEnabled;
+  replicaConfig.viewChangeProtocolEnabled = rp.viewChangeEnabled;
   replicaConfig.viewChangeTimerMillisec = rp.viewChangeTimeout;
+  replicaConfig.autoViewChangeEnabled = rp.autoViewChangeEnabled;
+  replicaConfig.autoViewChangeTimerMillisec = rp.autoViewChangeTimeout;
```

Comment:
We should also make sure those changes are replicated into the actual replica's configuration:
https://gitlab.eng.vmware.com/blockchain/vmwathena_blockchain/blob/master/concord/src/consensus/replica_imp.cpp#L214

https://gitlab.eng.vmware.com/blockchain/vmwathena_blockchain/blob/master/concord/src/consensus/replica_imp.cpp#L222